style.css

```diff
@@ -688,3 +688,126 @@ nav {
   list-style: none;
   display: flex;
 }
+
+.contact-me {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  background-color: #fff;
+}
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.contact-me h2 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  margin-top: 120px;
+  margin-left: 24px;
+  margin-right: 24px;
+  margin-bottom: 58px;
+}
+
+.contact-form input {
+  margin: 24px 0;
+  box-sizing: border-box;
+  padding: 15px 16px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.contact-form textarea {
+  width: 100%;
+  height: 114px;
+  background-color: #fbf8f7;
+  border: 0;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.contact-form,
+.contact-form-text,
+.contact-form-email {
+  border: none;
+  border-bottom: 2px solid #dbd8d7;
+}
+
+.contact-button {
+  display: flex;
+  flex-direction: row;
+  align-content: center;
+  align-items: center;
+  border: 0;
+  width: 140px;
+  height: 48px;
+  background: #ff6b00;
+  cursor: pointer;
+  transition: 0.5s;
+}
+
+input[type=submit] {
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-size: 17px;
+  font-weight: 700;
+  line-height: 24px;
+  padding: 12px 12px;
+  text-align: center;
+}
+
+.form-button {
+  margin-top: 24px;
+  margin-left: 160px;

```

Comment:
- [ ] I think it's better to use `margin: auto` instead of `margin-left: 160px` to center your button on the screen. Stick to the template is a project requirement.

<img width="929" alt="Screen Shot 2022-11-08 at 10 41 34 PM" src="https://user-images.githubusercontent.com/47673311/200717078-de3eb781-dd10-44b4-8c83-b93ffc98d9e9.png">